src/analyses/global_may_alias.h

```diff
@@ -54,6 +54,11 @@ class global_may_alias_domaint:public ai_domain_baset
     aliases.clear();
   }
 
+  void make_entry()
+  {
+    make_top();
+  }
+
```

Comment:
The global may alias analysis seems to have a bug in that ``make_bottom()`` and ``make_top()`` do the same thing and it does not have a separate bottom state. Fxing this might be something for a future pull request though.